src/mom5/ocean_core/ocean_sbc.F90

```diff
@@ -3967,7 +3969,7 @@ subroutine flux_adjust(Time, T_diag, Dens, Ext_mode, T_prog, Velocity, river, me
              do i=isc,iec
                 if(Grd%tmask(i,j,1) == 1.0) then 
                     ice_present = T_prog(index_temp)%field(i,j,1,tau) <= &
-                                    -0.0539*T_prog(index_salt)%field(i,j,1,tau)
+                                    -0.054*T_prog(index_salt)%field(i,j,1,tau)
```

Comment:
Constant changed here

Reply:
I love magic numbers. @russfiedler any comment on which is correct?

In the master branch there is an ACCESS ifdef code block afterwards that overwrites the ice_present value if `use_ioaice` is `.true.`

https://github.com/mom-ocean/MOM5/blob/master/src/mom5/ocean_core/ocean_sbc.F90#L4289

This defaults `.true.` so perhaps this can be changed back with impunity?

https://github.com/mom-ocean/MOM5/blob/43986e20236cc74b83cd5ee09ebd798f4da40824/src/mom5/ocean_access/auscom_ice_parameters.F90#L18

Reply:
@marshallward  I think this might have been done in order to get around rounding issues and makes it more likely that ice is detected.